### DISCUSSION TITLE
[Feature] Zoom in the photo at 2s hover on duplicates utility

### DISCUSSION THREAD

**Discussion by Hantick** [immich-app/immich#10547] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In the duplicates utility there should be an option to zoom in images to review which one we want to trash and which one to keep.
There could be a "check" button. I might try to develop it
![image](https://github.com/immich-app/immich/assets/36815427/0751e81d-8af7-4e9b-a627-9d37ce05aad3)

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#10173
